### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by yana-may25**

### CODE REVIEW

File: src/task2/task2.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+    function min(n, m)
```
Comment (MrOlegKravchenko) [inline src/task2/task2.js:1]: Please use one Function writing pattern:
```
func testFunc(n, m) {
    some actions
    ...
}
```

File: src/task2/task2.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    {
+        if (n===m) return "equal"
```
Comment (MrOlegKravchenko) [inline src/task2/task2.js:3]: also:
```
if (n===m) {
    ....
} else if (x===y) {
    ....
} else {
    ......
}
```

Comment (MrOlegKravchenko) [reply src/task2/task2.js:3]: No joke, my eyes are bleeding)
It's all in the topics, please don't read it through your fingers

File: src/task2/task2.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+        else
+       if (n<m) return n
+        else return m;
+
```
Comment (MrOlegKravchenko) [inline src/task2/task2.js:7]: But this function could return its one line answer right away.

File: src/task2/task2.js (hunk 2 above)
Comment (yana-may25) [reply src/task2/task2.js]: sorry I made u suffer. I'll try to pull my weight in the future

File: src/task2/task2.js [hunk 4]
```diff
@@ -1,8 +1,4 @@
...
-    {
-        if (n===m) return "equal"
-        else
-       if (n<m) return n
-        else return m;
-        
-    }
\ No newline at end of file
+function min(n, m) {
+    if (n === m) return "equal"
```
Comment (MrOlegKravchenko) [inline src/task2/task2.js:2]: Interesting behave, but min shouldn't return "equal"(

File: src/task2/task2.js [hunk 5]
```diff
@@ -1,8 +1,4 @@
...
+    else return (n < m ? n : m);
```
Comment (MrOlegKravchenko) [inline src/task2/task2.js:3]: Remember this ternary operator, you will use it plenty

File: src/task2/task2.js (hunk 2 above)
Comment (MrOlegKravchenko) [reply src/task2/task2.js]: image
And here we are again)

Comment (MrOlegKravchenko) [reply src/task2/task2.js]: and **always** end return value/expression with a semicolon

Comment (yana-may25) [reply src/task2/task2.js]: soooorry, I didn't get it back then, transparent as your comments might have been

**Review by MrOlegKravchenko — APPROVED** [review]

**Pull request merged**